common-money.lic

```diff
@@ -3,7 +3,9 @@
   Documentation: https://elanthipedia.play.net/Lich_script_development#common-money
 =end
 
-custom_require.call(%w[common common-travel])
+custom_require.call(%w[common common-travel drinfomon])
+
+#@settings = get_settings
```

Comment:
Was this left in by mistake?

Reply:
Was remarked out in later renditions. Will remove for final merge.